### ISSUE TITLE
Ability to set 'refresh_token' properties on a client

### ISSUE THREAD

**Issue by michaelgriffiths-mclaren** [alexkappa/terraform-provider-auth0#251]
### Description

We're facing issues with authentication in some of our clients which we've narrowed down to the `refresh_token.rotation_type` not being set to `rotating`.
It would be great if support could be added to the client resource to support setting those properties under the `refresh_token` field.

### New or Affected Resource(s)

* auth0_client

### Potential Terraform Configuration

```hcl
resource "auth0_client" "a_client" {
  name                = "A Client"
  ...
  refresh_token {
    rotation_type = "non-rotating"
    expiration_type = "non-expiring"
    leeway = 0
    token_lifetime = 2592000
  }
}
```

### References

Support added a few days ago to the underlying client:
* https://github.com/go-auth0/auth0/pull/120

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by rienafairefr** [comment]
Ive put a PR here https://github.com/terraform-providers/terraform-provider-auth0/pull/20 for that, though I'm now surprised there are two terraform-provider-auth0 repos, so maybe I didn't put that at the right place

**Comment by ninja-** [comment]
can we please merge this?

**Comment by maroux** [comment]
Looks like this was fixed by #255, except docs weren't updated in that PR. So - new issue for docs or ..?
